app/src/main/res/layout/fragment_referrals.xml

```diff
@@ -16,7 +16,7 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:clipToPadding="false"
-            android:paddingTop="?actionBarSize"
+            android:paddingTop="100dp"
```

Comment:
What was actionBarSize doing before? Is it just a design change or did the previous value not work for some reason?

Reply:
Had to adjust the padding here, the campaign bar was somehow laid out below the toolbar. Very strange, looked into different solutions and this worked best.